requirements.txt

```diff
@@ -18,7 +18,7 @@ prometheus-client>=0.13.1,<1
 fsspec==2022.1.0
 sentence-transformers>=2.2.0,<3
 wandb>=0.12.10,<0.13
-open-clip-torch>=1.0.1,<2.0.0
+open-clip-torch>=1.0.1,<=2.0.2
```

Comment:
Can you do < 3.0.0 instead?

Reply:
[8452f70](https://github.com/rom1504/clip-retrieval/pull/184/commits/8452f70c468f3aa5a6a3dc6c69b33e91b1d8df3d)